src/System.Globalization.Extensions/tests/Normalization/NormalizationAll.cs

```diff
@@ -14,192 +12,127 @@ public class NormalizationAll
-                    if (!bNFc5)
-                    {
-                        Console.WriteLine("'{0}' is marked as not normalized with form {1}", c5, normForm);
-                        failed = true;
-                    }
+                    Assert.True(isNormalized5, string.Format("'{0}' is marked as not normalized with form {1}", c5, normForm));
                     break;
 
                 case NormalizationForm.FormKC:
                     // c4 == NFKC(c1) == NFKC(c2) == NFKC(c3) == NFKC(c4) 
                     //    == NFKC(c5)
-                    failed |= CheckFailue(normForm, c4, strNFc1);
-                    failed |= CheckFailue(normForm, c4, strNFc2);
-                    failed |= CheckFailue(normForm, c4, strNFc3);
-                    failed |= CheckFailue(normForm, c4, strNFc4);
-                    failed |= CheckFailue(normForm, c4, strNFc5);
+                    CheckFailue(c4, normalized1);
+                    CheckFailue(c4, normalized2);
+                    CheckFailue(c4, normalized3);
+                    CheckFailue(c4, normalized4);
+                    CheckFailue(c4, normalized5);
 
                     // c4 is normalized to Form KC
-                    if (!bNFc4)
-                    {
-                        Console.WriteLine("'{0}' is marked as not normalized with form {1}", c4, normForm);
-                        failed = true;
-                    }
+                    Assert.True(isNormalized4, string.Format("'{0}' is marked as not normalized with form {1}", c4, normForm));
                     break;
 
                 case NormalizationForm.FormKD:
                     // c5 == NFKD(c1) == NFKD(c2) == NFKD(c3) == NFKD(c4) 
                     //    == NFKD(c5)
-                    failed |= CheckFailue(normForm, c5, strNFc1);
-                    failed |= CheckFailue(normForm, c5, strNFc2);
-                    failed |= CheckFailue(normForm, c5, strNFc3);
-                    failed |= CheckFailue(normForm, c5, strNFc4);
-                    failed |= CheckFailue(normForm, c5, strNFc5);
+                    CheckFailue(c5, normalized1);
+                    CheckFailue(c5, normalized2);
+                    CheckFailue(c5, normalized3);
+                    CheckFailue(c5, normalized4);
+                    CheckFailue(c5, normalized5);
 
                     // c5 is normalized to Form KD
-                    if (!bNFc5)
-                    {
-                        Console.WriteLine("'{0}' is marked as not normalized with form {1}", c5, normForm);
-                        failed = true;
-                    }
+                    Assert.True(isNormalized5, string.Format("'{0}' is marked as not normalized with form {1}", c5, normForm));
                     break;
             }
-            return failed;
         }
 
-        private static bool CheckFailue(NormalizationForm normForm, string c, string cForm)
+        private static void CheckFailue(string c, string cForm)
```

Comment:
CheckFailure is mispelled.  It could also use a more descriptive name, like AssertEqualsForm.